posts/2024-02-15-Windows-7.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Updated baseline standards for Windows targets"
+author: Chris Denton on behalf of the Compiler Team
+---
+
+The minimum requirements for Tier 1 toolchains targeting Windows will increase with the 1.77 release (scheduled for March 21, 2024).
+Windows 10 will now be the minimum supported version for the `*-pc-windows-*` targets.
+These requirements apply both to the Rust toolchain itself and to binaries produced by Rust.
+
+Two new targets have been added with Windows 7 as their baseline: `x86_64-win7-windows-msvc` and `i686-win7-windows-msvc`.
+They are starting as Tier 3 targets, meaning that the Rust codebase has support for them but we don't build or test them automatically.
+Therefore binaries aren't distributed via rustup.
```

Comment:
nit: sentence fragment

I think it would be better to word this in a way that gets to what users probably want to know, ie:

```suggestion
Once these targets reach Tier 2 status, they will be available to use via rustup.
```

Unless we don't think Win7 will ever be Tier 2 status for whatever reason.